continuationsPlugin/src/main/scala/continuations/CallsContinuationResumeWith.scala

```diff
@@ -24,8 +32,13 @@ private[continuations] object CallsContinuationResumeWith extends Trees:
       tree
         .rhs
         .filterSubTrees {
-          case Inlined(fun, _, _) =>
-            fun.denot.matches(suspendContinuationMethod.symbol)
+          case Inlined(call, _, _) =>
+            val isSuspendContinuation = call.denot.matches(suspendContinuationMethod.symbol)
+            if isSuspendContinuation && hasNestedContinuation(call) then
+              report.error(
```

Comment:
long shot but I guess there is no way to collect those errors for the tests?

Reply:
I haven't figured out how to do it yet. It's probably in the compiler tests in the dotty repo if you want to look.